src/main/java/com/xpandit/plugins/xrayjenkins/Utils/BuilderUtils.java

```diff
@@ -31,6 +34,44 @@ public static boolean isSupportedJobType(Class<? extends AbstractProject> jobTyp
                 || MavenModuleSet.class.isAssignableFrom(jobType);
     }
 
+    /**
+     * Utility method to check if the endpoint supports glob expressions
+     * @param endpointValue the endpoint value
+     * @return <code>true</code> if the endpoint supports glob expressions , <code>false </code> otherwise
+     */
+    public static boolean isGlobExpressionsSupported(Endpoint endpointValue) {
+        return (Endpoint.JUNIT.equals(endpointValue)
+                || Endpoint.NUNIT.equals(endpointValue)
+                || Endpoint.TESTNG.equals(endpointValue)
+                || Endpoint.ROBOT.equals(endpointValue)
+                || Endpoint.XUNIT.equals(endpointValue)
+                || Endpoint.JUNIT_MULTIPART.equals(endpointValue)
+                || Endpoint.ROBOT_MULTIPART.equals(endpointValue)
+                || Endpoint.TESTNG_MULTIPART.equals(endpointValue)
+                || Endpoint.NUNIT_MULTIPART.equals(endpointValue)
+                || Endpoint.XUNIT_MULTIPART.equals(endpointValue));
+    }
+
+    /**
+     * Utility method to get the generic endpoint from the corresponding multipart endpoint.
+     * In example, for the junit multipart url: /junit/multipart
+     *             we extract the generic url: /junit
+     * @param multipartEndpointSuffix the corresponding multipart endpoint suffix
+     * @return the new generic endpoint
+     */
+    public static Endpoint getGenericEndpointFromMultipartSuffix(String multipartEndpointSuffix) {
+            String[] multipartEndpointSuffixSeparated = multipartEndpointSuffix.split(SLASH_SEPARATED_REGEX);
+
+            //Build the suffix of the generic endpoint by extracting the endpoint name from the multipart one and adding a slash before.
+            String genericEndpointName = "/" + multipartEndpointSuffixSeparated[1];
```

Comment:
To avoid an _OutOfBoundException_, please check the size if the array in line 63

Reply:
In this case, you could throw an exception and log the Endpoint received as input